src/directives/layertree.js

```diff
@@ -4,15 +4,25 @@ goog.provide('ngeo.layertreeDirective');
 goog.require('ngeo');
 
 
+/**
+ * Unique template path for the layertree;
+ * @type {string|undefined}
+ */
+ngeo.layertreeTemplateUrlValue = undefined;
```

Comment:
It's hard to explain. I'll speak about that during monday's Scrum. However, I'll retry:

 - Something define the tree, or root node, with a ngeoLayertreeTemplateUrl.
 - This template represente a node. a node is another tree that defines a ngeoLayertreeTemplateUrl.

These both points generate trees from differents files.

You'll get a path problem. Relatives URI starts from differents directories in 'examples' or  examples-hosted.